package.json

```diff
@@ -18,7 +18,7 @@
   "dependencies": {
     "semver": "^7.3.2",
     "shelljs": "^0.8.3",
-    "typescript": "^4.5.5"
+    "typescript": "4.7.0-beta"
```

Comment:
I'd prefer `next` so that `npm install` might (?) do the right thing to update to the very latest version.